packages/keyring-api/src/trx/types.ts

```diff
@@ -0,0 +1,54 @@
+import { object } from '@metamask/keyring-utils';
+import type { Infer } from '@metamask/superstruct';
+import { array, enums, literal, nonempty } from '@metamask/superstruct';
+import { definePattern } from '@metamask/utils';
+
+import {
+  CaipChainIdStruct,
+  KeyringAccountStruct,
+  TronAccountType,
+} from '../api';
+
+/**
+ * TRON addresses are Base58-encoded strings that are exactly 34 characters long
+ * and start with the letter 'T'.
+ * Mainnet addresses begin with 'T'.
```

Comment:
Can you also document the pattern of testnet account addresses?

Reply:
Removed that last line as it was misleading. I looked into the testnets and they all follow the same standard 👍